### DISCUSSION TITLE
Help please i want add new language to foriceboard

### DISCUSSION THREAD

**Discussion by UdulaDamsara** [florisboard/florisboard#2056] [Q&A]
How to add new language to this keyboard 🤔

**Comment by patrickgold** [comment]
Which language to you want to translate FlorisBoard into? Have a look at the [Crowdin project](https://crowdin.com/project/florisboard), if the language is listed there you can help finishing it, it it isn't listed I will add it.

**Reply by patrickgold** [reply]
Or do you ask for a language support for auto-correction? In this case you'll need to wait for some time, as I am currently beginning to experiment with adding support for this.

**Comment by UdulaDamsara** [comment]
> Which language to you want to translate FlorisBoard into? Have a look at the [Crowdin project](https://crowdin.com/project/florisboard), if the language is listed there you can help finishing it, it it isn't listed I will add it.

No..no..
I want add new language for keyboard keys..
